src/frontend/hooks/server/maps.ts

```diff
@@ -27,23 +27,33 @@ export type CustomMapInfo = v.InferOutput<typeof CustomMapInfoSchema>;
 
 const {useRefreshToken, useRefreshTokenActions} = createRefreshTokenStore();
 
-export function useMapStyleJsonUrl() {
-  const api = useClientApi();
+export function useCustomMapStyleUrl() {
```

Comment:
I think the old name is more accurate in terms of what this does (the url doesn't always point to a custom map style)

```suggestion
export function useMapStyleJsonUrl() {
```